lms/static/sass/elements/_progress-circle.scss

```diff
@@ -0,0 +1,59 @@
+.program-progress {
+    width: 300px;
+    margin: 0 auto 30px;
+
+    @media(min-width: $bp-screen-md) {
+        margin-left: 0;
+    }
+}
+
+.progress-heading {
+    color: $blue-d1;
```

Comment:
It would be good to make these colors into variables, so that it is easy for a theme to override them.